package ch14_exception;

public class Testfor {
	public static void main(String args[]){
		/*for(int i=1;i<=10;i++){
			System.out.print(i+" ");
		}//for end  */
		
		
		//난수구하기
	for(int i=1;i<=1;i++){
		int a=(int)(Math.random()*(10+1));
				System.out.print(i+"");
	}
	
	}//main end
}//class end  
		
		
